src/ch02-01-initializing-bastion.md

```diff
@@ -0,0 +1,125 @@
+# Initializing Bastion
+
+## Initialize
+
+Bastion require to be initialized. You can init it with the default configuration or with a custom one. A custom configuration will allow you to override the `Backtraces`\* system
+
+Enabled by default, a custom configuration will allow you to override it and enable or disable the `Backtraces`.
+
+- Default configuration
+
+```rs
+use bastion::prelude::*;
+
+fn main() {
+    Bastion::init();
+
+    // Some stuff have to be write after... but not yet!
+}
+```
+
+- Custom configuration, we are hidding the backtraces here, you can still use `show_backtraces();` to enable it.
+
+```rs
+use bastion::prelude::*;
+
+fn main() {
+    let config = Config::new().hide_backtraces();
+    Bastion::init_with(config);
+
+    // Some stuff have to be write after... but not yet!
+}
+```
+
+## Start
+
+After initialize bastion you will have to start it by using `Bastion::start();`.
+
+```rs
+use bastion::prelude::*;
+
+fn main() {
+    Bastion::init();
+    Bastion::start();
+
+    // Some stuff have to be write after... but not yet!
+}
+```
+
+## Block
+
+As with any application, there will be a point where you want to stop it running. But this time might not occur in your `main()` function. This can happen elsewhere in your code.
```

Comment:
```suggestion
As with any application, there will be a point where you want it to stop. But this time might not occur in your `main()` function. This can happen anywhere in your code.
```